### DISCUSSION TITLE
Want to terminate CI pipeline if any fatal error comes

### DISCUSSION THREAD

**Discussion by AmandeepSaxena** [cloudflare/pint#871] [General]
@prymitive I am using pint in ci [referring](https://cloudflare.github.io/pint/#usage)
And after testing I found it is generating a report, in place of that what I want is to get error while pipeline build ( like ./pint lint rules.yaml works) and get error in github workflow check and if any fatal error comes that PR will be failed to get merge. How can I do that? I am confused if I can do any configuration for that. Please help me here.

**Comment by prymitive** [comment]
`pint ci` exits with a non-zero code if there are `bug` level problems reported.
You can set it to a lower level if you want by passing `fail-on` (pint ci --fail-on=warning

```
$ ./pint ci -h
[..]
   --fail-on value, -w value      Exit with non-zero code if there are problems with given severity (or higher) detected (default: "bug")
[...]
```
